backend/core/scripts/listings-importer.ts

```diff
@@ -132,9 +130,10 @@ export async function importListing(
 
   await uploadAmiCharts(listing.units)
 
-  // Replace each unit's unitType string with a foreign-key reference to the corresponding row in
-  // the unit_types table.
+  // Replace each unit's and unitsSummary's unitType string with a foreign-key reference to the corresponding
+  // row in the unit_types table.
   await linkToUnitTypes(listing.units)
```

Comment:
I think we can delete this line, too? And switch `linkToUnitTypes` to accept an array of `UnitsSummary`

Reply:
Oh actually never mind, the other callers of this `importListing` function may supply `units` instead of summaries, so let's keep it!